src/iris/ui/static/js/iris.js

```diff
@@ -129,6 +129,7 @@ iris = {
       notificationTemplateSource: $('#plan-notification-template').html(),
       stepTemplateSource: $('#plan-step-template').html(),
       trackingTemplateSource: $('#plan-tracking-notification-template').html(),
+      testPlanSource: $('#test-plan-template').html(),
```

Comment:
maybe add `Template` to this variable name to keep it consistent with the others